README.md

```diff
@@ -1,90 +1,96 @@
 
 **2. "FixedHeight" method.**
 
-Resizes the image to the desired size while maintaining the aspect ratio without cropping. Width of thumbnail will be calculated according aspect ratio. 
+Resizes the image to the desired size while maintaining the aspect ratio without cropping. The width of thumbnail will be calculated according to the aspect ratio.
 
 **3. "FixedWidth" method.**
 
-Resizes the image to the desired width while maintaining the aspect ratio.  If aspect ratio of original image and the thumbnail doesn't match then white space will be filled with color of background.
+Resizes the image to the desired width while maintaining the aspect ratio.  If the aspect ratio of the original image and the thumbnail don't match , then the white space will be colored with the background color.
 
 **4. "Crop" method.**
 
-Resize the image until one of the sides will match to the given dimensions while maintaining the aspect ratio. Part of the image will be cut off. The thumbnail will be resized and cropped the anchor position. You can specify an anchor position in the o'sption Anchor Position: TopLeft, TopCenter, TopRight, CenterLeft, Center, CenterRight, BottomLeft, BottomCenter, BottomRight.
+This method resizes the image until one of the sides matches the given dimensions while maintaining the aspect ratio. A part of the image will be cut off. The thumbnail will be resized and cropped according to the anchor position. You can specify an anchor position in the o'sption Anchor Position: TopLeft, TopCenter, TopRight, CenterLeft, Center, CenterRight, BottomLeft, BottomCenter, BottomRight.
 
 Now you can use the two thumbnails wherever you want to display a smaller version of the original image.
 You can do it, just add a needed suffix (grande or medium) to product image url.
 
-# Running the task
+## Running Multiple tasks
 
 You can select multiple task and run them in the task list or run just one task from the task blade.
 
 ![image](https://user-images.githubusercontent.com/20122385/36430796-b68279c6-165e-11e8-8e02-ae75e7a11f66.png)
 
 Available modes:
 
-* Regenerate - thumbnails will be generated for all images. 
-* Process changes - tries to find changed or added images and generates thumbnails only for them. Also will detect added options. Already generated thumbnails for removed options will not be deleted. 
+* **Regenerate** - thumbnails will be generated for all images.
+* **Process changes** - tries to find changed or added images and generates thumbnails only for them. Also will detect added options. Thumbnails generated for removed options will not be deleted.
+
+## Installation
 
-# Installation
 Installing the module:
+
 * Automatically: in VC Manager go to Configuration -> Modules -> Image tools module -> Install
```

Comment:
Automatically: in VC Manager go to More -> Modules -> Available (n) -> Image tools module -> Install